src/styles/variables.scss

```diff
@@ -0,0 +1,20 @@
+$font-family: Arial, sans-serif;
+$cell-size: 100px;
+$font-size: 30px;
+$cell-color: #eee;
+$active-cellColor: #ffbfcb;
+$gap: 1px;
+$border-width: 1px;
+$padding: 10px;
+$columns: 7;
+
+// $days:
+//   'mon' 0,
+//   'tue' 1,
+//   'wed' 2,
+//   'thu' 3,
+//   'fri' 4,
+//   'sat' 5,
+//   'sun' 6;
+// src/styles/variables.scss
+//  10:6  ✖  Expected single space after ":"  scss/dollar-variable-colon-space-after
```

Comment:
consider removing redundant comments

Reply:
when I created the $days variable, I encountered an error after running the tests (npm t), even though there wasn't any extra space. As a result, I had to write all the values directly in index.scss